_data/dev-docs/guides/p2p_network.md

```diff
@@ -211,20 +211,20 @@ This allows the IBD node to detect potential forks in its network, by validating
 TODO: Explain conflict resolution after finding a fork
 {/{% comment %}}
 
-![Second GETB Message Sent During IBD](/img/dev/en-ibd-getb2.svg)
+![Second GETB Message Sent During IBD](/dev-docs/img/dev/en-ibd-getb2.svg)
```

Comment:
@stdevHan please all images go in `img/` no `dev-docs` path prefix and no `dev` subfolder.